frontend/static/js/joelFuncs.js

```diff
@@ -0,0 +1,174 @@
+// https://github.com/jo3-l/yagfuncdata
+var joelFuncs = {
```

Comment:
```suggestion
var joelFuncs = {
var yagFuncs = {
```
This should be a more descriptive name IMO